libs/fundamental-redirects/index.js

```diff
@@ -1147,6 +1147,37 @@ for (const [pattern, path] of [
   );
 }
 
+function simpleRedirect(
+  pattern,
+  replacement,
+  { permanent = true, colonToSlash = false } = {}
+) {
+  return (path) => {
+    const match = pattern.exec(path);
+    if (match === null) {
+      return null;
+    }
+    const status = permanent ? 301 : 302;
+    let url = path.replace(pattern, replacement);
+    if (colonToSlash) {
+      url = url.replace(/:/g, "/");
+    }
+    return { url, status };
+  };
+}
+
+const SIMPLE_REDIRECT_PATTERNS = [
+  simpleRedirect(
+    /\/docs\/Core_JavaScript_1.5_?/,
```

Comment:
to really work with locales this should be a `localeRedirect` or we mimic it in there. Most important would be to start the regex with `^` to be sure that this will not catch a weird slug where the `/docs/...` is somewhere repeated like `/en-US/docs/never/use/docs/Core_JavaScript_1.5_Reference:Statements:block`

Reply:
I switched to `localeRedirect()` now. But extended it a bit.